frontend/packages/console-app/src/components/guided-tours/GuidedTourDrawer.scss

```diff
@@ -0,0 +1,12 @@
+.co-guided-tour-drawer {
+  &__panel-content {
+    overflow-y: auto !important;
+    z-index: auto;
+  }
+
+  &__subtitle {
+    display: inline;
```

Comment:
Probably good to use `inline-block` to force the duration to wrap as a single unit when the title is very long.

Reply:
done